modules/microshift-accessing-cluster-open-firewall.adoc

```diff
@@ -8,7 +8,7 @@
 [id="microshift-accessing-cluster-open-firewall_{context}"]
 = Opening the firewall for remote access to the {product-title} cluster
 
-Use the following procedure to open the firewall so that a remote user can access the {product-title} cluster. Your account must have cluster admin privileges. This procedure must be completed before a workstation user can access the cluster remotely.
+Use the following procedure to open the firewall so that a remote user can access the {product-title} cluster. This procedure must be completed before a workstation user can access the cluster remotely. Your account must have cluster administration privileges.
```

Comment:
`This procedure must be completed before a workstation user can access the cluster remotely. Your account must have cluster administration privileges.` These sound like prerequisites to me.